### PR TITLE
fix: Critical threshold is optional

### PR DESCRIPTION
**Pull request by Fallenstedt**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
https://issues.newrelic.com/browse/NR-54573

NRQL Alert Conditions will _soon_ no longer require critical threshold. NRQL Alert Conditions will allow critical thresholds to be optional.

* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
  
  Upcoming UI changes to the Condition Builder. Users will soon have an updated threshold building experience. They will be able to create NRQL Alert conditions with only warning thresholds. 
  
Screen Shot 2022-11-09 at 3 17 11 PM

  
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by bradleycamacho** [comment]
Hi @Fallenstedt, thanks for the edits! I'm tagging @homelessbirds to review this.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx [hunk 1]
```diff
@@ -121,7 +121,11 @@ Details about other functionality and rules:
           </td>
 
           <td>
+<<<<<<< HEAD
```
Comment (Fallenstedt) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition.mdx:124]: 🔧  Woops. I'll fix this here, just a sec.

**Comment by Fallenstedt** [comment]
I'm not sure what happened with the Gatsby build here. 😬

**Comment by Fallenstedt** [comment]
@homelessbirds, DCON intends to make the critical term optional on November 29th. Please let me know if there are any additional changes or information you need from me for this docs update.

cc: @rogernewrelic

**Comment by austin-schaefer** [comment]
@homelessbirds : Can you take a look at this one?

**Review by homelessbirds — APPROVED** [review]

**Comment by homelessbirds** [comment]
Manually merging after consulting with Liz re: cache build.

**Pull request merged**